Add location module information. (#4599)
braze-inc/braze-docs: _docs/_developer_guide/platform_integration_guides/android/initial_sdk_setup/android_sdk_integration.md

Add the `android-sdk-ui` dependency to your app's `build.gradle`. 

> If you are using any location or Braze Geofence functionality, also include `android-sdk-location` to your app's `build.gradle`.

Reviewer: ```suggestion
If you are using any location or Braze Geofence functionality, also include `android-sdk-location` in your app's `build.gradle`.
```

> If you are using any of the location or geofence functionality and also using and SDK version of `24.0.0` or greater, also include `android-sdk-location` to your app's `build.gradle`.

Reviewer: ```suggestion
If you are using any location or Braze Geofence functionality, also include `android-sdk-location` to your app's `build.gradle`.
```
